alpha/template/semver/semver.go

```diff
@@ -61,12 +54,16 @@ func (t Template) Render(ctx context.Context) (*declcfg.DeclarativeConfig, error
 	return &out, nil
 }
 
-func buildBundleList(bundles *[]semverTemplateBundleEntry, dict *map[string]struct{}) {
-	for _, b := range *bundles {
-		if _, ok := (*dict)[b.Image]; !ok {
-			(*dict)[b.Image] = struct{}{}
+func buildBundleList(t semverTemplate) map[string]string {
+	dict := make(map[string]string)
+	for _, bl := range []semverTemplateChannelBundles{t.Candidate, t.Fast, t.Stable} {
```

Comment:
nit: wdyt about making the making the semverTemplateChannelBundles list a parameter? Before we could see which channel templates were being used to create the bundle list at the top level.  Might also make the method more flexible? (though given that it isn't exported - maybe not...XD).